Lib/test/_test_embed_structseq.py

```diff
@@ -1,29 +1,28 @@
 import sys
 import types
-import unittest
 
 
 # bpo-46417: Test that structseq types used by the sys module are still
 # valid when Py_Finalize()/Py_Initialize() are called multiple times.
-class TestStructSeq(unittest.TestCase):
+class TestStructSeq:
     # test PyTypeObject members
-    def check_structseq(self, obj_type):
+    def _check_structseq(self, obj_type):
         # ob_refcnt
-        self.assertGreaterEqual(sys.getrefcount(obj_type), 1)
+        assert(sys.getrefcount(obj_type) > 1)
         # tp_base
-        self.assertTrue(issubclass(obj_type, tuple))
+        assert(issubclass(obj_type, tuple))
         # tp_bases
-        self.assertEqual(obj_type.__bases__, (tuple,))
+        assert(obj_type.__bases__ == (tuple,))
         # tp_dict
-        self.assertIsInstance(obj_type.__dict__, types.MappingProxyType)
+        assert(isinstance(obj_type.__dict__, types.MappingProxyType))
         # tp_mro
-        self.assertEqual(obj_type.__mro__, (obj_type, tuple, object))
+        assert(obj_type.__mro__ == (obj_type, tuple, object))
         # tp_name
-        self.assertIsInstance(type.__name__, str)
+        assert(isinstance(type.__name__, str))
         # tp_subclasses
-        self.assertEqual(obj_type.__subclasses__(), [])
+        assert(obj_type.__subclasses__() == [])
 
-    def test_sys_attrs(self):
+    def sys_attrs(self):
```

Comment:
Why not keep the `test_foo` method naming? It seems nice to keep a verb in the test method names.

Reply:
I removed it since it was not using the unittest framework anymore, but you're right we can keep the naming convention. Renamed!